### PR TITLE
Feature/20 database model for the session

### LINKED ISSUE #20: Implement the database model for the Session
Asana task: https://app.asana.com/0/1204166279674278/1208290141948825/f

### PR DESCRIPTION
**Pull request by ifirmawan**
## TODO/DONE

- [x] Models & Migration
- [x] ENUM Sector
- [x] Database Seeder
- [x] Unit tests

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208290141948825

### CODE REVIEW

File: backend/api/v1/v1_sessions/constants.py
Comment (dedenbangkit) [inline backend/api/v1/v1_sessions/constants.py:1]: Hi @ifirmawan please replace with this (you might need to remake-the-migration)

Policy coherence
International fair agreements
Equality, inclusion & diversity
Human rights
Global Citizen & Global Engagement
Collaborate on sustainable development
Promoting peace and stabilising conflict
Nature, environment & climate justice
Fair distribution & access to basic amenities
Humanitarian emergency aid

File: backend/api/v1/v1_users/management/commands/fake_users_seeder.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            countries_json = json.load(countries_json)
+            countries = [
+                c["alpha-2"]
+                for c in countries_json
+            ]
+        repeat = options.get("repeat")
+
+        for r in range(repeat):
+            gender = self.fake_gender()
+            sex_code = {
```
Comment (dedenbangkit) [inline backend/api/v1/v1_users/management/commands/fake_users_seeder.py]: gender_code

File: backend/api/v1/v1_users/management/commands/fake_users_seeder.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                1: "M",
+                2: "F",
+                3: random.choice(["M", "F"])
+            }
+            sex = sex_code[gender]
```
Comment (dedenbangkit) [inline backend/api/v1/v1_users/management/commands/fake_users_seeder.py]: remove

File: backend/api/v1/v1_users/management/commands/fake_users_seeder.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            profile = fake.simple_profile(sex=sex)
+            country = random.choice(countries)
+            SystemUser.objects.create_user(
+                email=profile["mail"],
+                password="Test1234",
+                full_name=profile["name"],
+                date_joined=fake.date_this_year(),
+                account_purpose=self.fake_account_purpose(),
+                country=country,
+                gender=gender,
```
Comment (dedenbangkit) [inline backend/api/v1/v1_users/management/commands/fake_users_seeder.py:53]: gender_code[gender]

**Review by dedenbangkit — CHANGES_REQUESTED** [review]

File: backend/api/v1/v1_sessions/constants.py
Comment (ifirmawan) [reply backend/api/v1/v1_sessions/constants.py:1]: Ok mas, on it

**Review by dedenbangkit — APPROVED** [review]

**Pull request merged**